src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types';
+
+type Props = {
+  userInfo: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ userInfo }) => {
+  return (
+    <a className="UserInfo" href="mailto:Sincere@april.biz">
```

Comment:
The email address is hardcoded as 'Sincere@april.biz'. If the intention is to use the user's actual email, consider using `userInfo.email` instead to make the component dynamic and accurate.